test/spec/geo/ui/time_slider.spec.js

```diff
@@ -63,6 +63,20 @@ describe('cdb.geo.ui.TimeSlider', function() {
       });
     });
 
+    describe("given a step that spans more than 48 hours", function() {
+      beforeEach(function() {
+        var start = 1423699205000;
+        var end = 1424649534000;
+        view.torqueLayer.getTimeBounds = function(){return {start: start, end: end}};
+        view.torqueLayer.options = {steps: 5};
+        formatter = view.formatterForRange.bind(view)(start, end);
+      });
+      it("should return a formatter function that defines a two-date range", function() {
+        var moment = new Date(1424269402400);
+        expect(formatter.bind(view)(moment)).toEqual("Feb 18 - Feb 20");
```

Comment:
That's great to know, thanks, I'm still getting used to Javascript's context model. Will correct it now.